file_optical.c

```diff
@@ -1752,6 +1752,22 @@ static int fbo_handle_cmd(struct tcmu_device *dev, struct tcmulib_cmd *cmd)
 		return fbo_emulate_mechanism_status(dev, cdb, iovec, iov_cnt,
 						    sense);
 		break;
+	case RESERVE_10:
+		if (cdb[1] & 3)
+			return tcmu_set_sense_data(sense, ILLEGAL_REQUEST,
+						   ASC_INVALID_FIELD_IN_CDB,
+						   NULL);
+		else
+			return SAM_STAT_GOOD;
+		break;
+	case RELEASE_10:
+		if (cdb[1] & 3)
+			return tcmu_set_sense_data(sense, ILLEGAL_REQUEST,
+						   ASC_INVALID_FIELD_IN_CDB,
+						   NULL);
+		else
+			return SAM_STAT_GOOD;
+		break;
 	default:
```

Comment:
Did part of your commit get dropped? IIs there supposed to be more code for this commit? All I see is the above code. I did not see anything like some code checking for conflicts and existing reservations.